docs/metrics.md

```diff
@@ -111,6 +114,33 @@ The **methods** are:
    1. `exemptTab`: the user has decided to disable the proxy for this tab.
 
 
+### scalar Registration and Recording
```

Comment:
For this section you might just wish to link out to the [in-tree Scalar docs](https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/collection/scalars.html) and force us to keep them updated for you : )